test/integration/model/multiple_lights/model.sdf

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<sdf version="1.8">
+  <light name="multiple_lights1" type="directional">
```

Comment:
Can you add a comment stating that this is an invalid file? Users might look through this directory to find example usages of SDFormat, so we should probably warn them when files are invalid.

Reply:
I was under the impression that we are deeming all such files invalid whether they are being included or not. i.e, one file should always represent one model, light, or actor. Another use case is adding a model into Gazebo using `gz model`, but even in this case, a single model is expected and it's name and pose can be overriden.

Reply:
Ahh, got it. Makes sense

Reply:
This also made me think, we should probably add some logic to the parser to handle the case where a model is being loaded directly (not included) and has multiple models/lights/actors. This is commonly done via `sdf::Root::Load` or `sdf::Root::LoadSdfString`, but the core of the parsing is still done in `sdf::readXml` in `parser.cc`.

Reply:
I missed that last comment, so I'll add some more tests for it. But I have updated the sdf root spec in the latest commit. I think I have the `required` syntax correct.